chain-signatures/crypto/src/kdf.rs

```diff
@@ -28,6 +29,7 @@ impl Chain {
             Chain::Ethereum => "0x1",
             Chain::Solana => "0x800001f5",
             Chain::Bitcoin => "bip122:000000000019d6689c085ae165831e93",
+            Chain::Hydration => "polkadot:2034",
```

Comment:
`2034` is not a CAIP-2 identifier, it's a `ParaId` that identifies it inside of Polkadot network.
CAIP 2 is using the `genesis hash`. OS it should be `polkadot:afdc188f45c71dacbaa0b62e16a91f72`.

Reply:
Then we need to change it there @esaminu and fix here